internal/pkg/describe/env_test.go

```diff
@@ -4,24 +4,29 @@
 package describe
 
 import (
+	"errors"
+	"fmt"
+	"testing"
+
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/archer"
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/deploy/cloudformation/stack"
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/describe/mocks"
 	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/require"
-	"testing"
 )
 
 type envDescriberMocks struct {
-	mockStoreService   *mocks.MockstoreSvc
+	mockStoreService         *mocks.MockstoreSvc
 	mockResourceGroupsClient *mocks.MockresourceGroupsClient
 }
 
 func TestEnvDescriber_FilterAppsForEnv(t *testing.T) {
 	const (
-		badParsedStack  = "badMockArn"
-		testParsedStack1 = "phonetool-test-app1"
-		testParsedStack2 = "phonetool-prod-app2"
+		testARN1         = "arn:aws:cloudformation:us-west-2:123456789012:stack/testProject-testEnv1-testApp1/6d75d1g0-8b1a-11ea-b358-06c1882c17fd"
+		testARN2         = "arn:aws:cloudformation:us-west-2:123456789012:stack/testProject-testEnv1-testApp3/7d75d1f0-8c1a-11ea-b358-06c1882c17fc"
```

Comment:
nit: it's a little confusing to have this const be called `testARN2` but the app name `testApp3` -- it's not a big deal but does add a bit more cognitive load.